packages/server/lambdas/alfresco/delete-folder.ts

```diff
@@ -61,13 +60,9 @@ export async function handleRequest(event: ALBEvent): Promise<ALBResult | undefi
     validateWriteUser(user, writeRole);
 
     const headers = (await getAlfrescoOptions(user.uid)).headers;
-    const requestOptions = folderDeleteRequestBuilder(headers) as RequestInit;
-
-    const alfrescoResult = await deleteFolder(requestOptions, nodeId);
-    if (!alfrescoResult) {
-      throw new RataExtraLambdaError('Error deleting folder from Alfresco', 404);
-    }
+    const requestOptions = (await folderDeleteRequestBuilder(headers)) as RequestInit;
 
```

Comment:
Could I ask why did you remove conditional check`!alfrescoResult` before deleting database record?

Reply:
Good catch. I remember the conditional causing some issues when deleting folder and I must have forgotten to apply it back.